### PR TITLE
Update Design Standards

### PR DESCRIPTION
**Pull request by okkays**
Implements the redesign of the [design standards page](https://www.ada.gov/law-and-regs/design-standards/).

- 🌎 The design standards don't currently include images
- ⛔ This makes it hard to see at-a-glance what document you're
  downloading or page you're going to.
- ✅ This PR adds images and groups standards with guidance to make documents distinguish to find at-a-glance.

Before:

Previous design standards (gif demo - see federalist preview link)

After:

New design standards (gif demo - see federalist preview link)

### CODE REVIEW

**Review by nguyen-tina — COMMENTED** [review]
This is mostly looking good – one nit: Is it possible to make the icon + download text flush to the line above? There's currently 16px margin that's not needed.

**Comment by nguyen-tina** [comment]
Also, strangely, the icon for the Guidance on the 2010 ADA Standards for Accessible Design is smaller than the other two icons...

**Comment by c2nelson** [comment]
I think this looks great.  I agree with @nguyen-tina re tweaking the margin.

**Comment by okkays** [comment]
> This is mostly looking good – one nit: Is it possible to make the icon + download text flush to the line above? There's currently 16px margin that's not needed.

Margin removed, replaced with an 8px gap that should only appear between the icon and text (and anything else we'd ever decide to add to the link)

> Also, strangely, the icon for the Guidance on the 2010 ADA Standards for Accessible Design is smaller than the other two icons...

This was a consequence of flexbox thinking that the text was more important than the icon - nice catch, fixed!

File: _assets/sass/custom/_buttons.scss [hunk 1]
```diff
@@ -35,10 +35,11 @@
 .crt-page--downloadpdf-link {
   align-items: baseline;
   display: inline-flex;
-}
+  gap: 8px;
```
Comment (kewitham) [inline _assets/sass/custom/_buttons.scss:38]: I didn't know this was a thing! Super useful!

Comment (okkays) [reply _assets/sass/custom/_buttons.scss:38]: 😄 [I definitely recommend this guide](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) - it's definitely worth skimming the pictures, I go back to it pretty much every time I use flex.

Comment (nguyen-tina) [reply _assets/sass/custom/_buttons.scss:38]: these pics are so useful

File: _assets/sass/custom/_page.scss [hunk 2]
```diff
@@ -66,3 +66,31 @@
...
+  .document {
+    width: 100%;
+
+    img {
+      border: 1px solid lightgray;
+      display: block;
+      margin-bottom: 1em;
+      max-width: 50%;
+      @include at-media-max(mobile-lg) {
+        max-width: 100%;
```
[the remark this answers is not in the document]
Comment (nguyen-tina) [reply _assets/sass/custom/_page.scss:92]: Agreed that keeping them at 50% is good for the user!

[the remark this answers is not in the document]
Comment (okkays) [reply _assets/sass/custom/_page.scss]: Done!

Had a quick thought about how it'd look center-aligned, but left still looks better IMO

current:

screenshot with left-aligned cover photo

centered:

screenshot with center-aligned cover photo

[the remark this answers is not in the document]
Comment (nguyen-tina) [reply _assets/sass/custom/_page.scss]: great job on moving quickly Kevin! 

they both look good, and at this point it's preference – I think we should keep it consistent with what it is on desktop, even though the center-alignment is so tempting!

**Comment by okkays** [comment]
@vwilliamsn Related content removed!

**Pull request merged**